lib/platform/gitlab/types.ts

```diff
@@ -10,13 +10,18 @@ export interface GitlabComment {
 
 export type MergeMethod = 'merge' | 'rebase_merge' | 'ff';
 
+export type Namespace = {
+  path: string;
+};
+
 export type RepoResponse = {
   archived: boolean;
   mirror: boolean;
   default_branch: string;
   empty_repo: boolean;
   http_url_to_repo: string;
   forked_from_project: boolean;
+  namespace: Namespace;
```

Comment:
This should be optional, if it is optional. 
```suggestion
  namespace?: Namespace;
```